bot/exts/evergreen/conversationstarters.py

```diff
@@ -24,6 +23,10 @@
     # All the allowed channels that the ".topic" command is allowed to be executed in.
     ALL_ALLOWED_CHANNELS = [channel_id for channel_id in PY_TOPICS.keys()] + list(WHITELISTED_CHANNELS)
 
+# Putting all topics into one dictionary and shuffling lists to reduce same-topic repetitions.
+all_topics = {'default': STARTERS, **PY_TOPICS}
```

Comment:
```suggestion
ALL_TOPICS = {'default': STARTERS, **PY_TOPICS}
```
This should be uppercased to signify that it should be treated as a constant,